.gitignore

```diff
@@ -1,297 +1,4 @@
 # macOS
 .DS_Store
 
-# all PyCharm settings
```

Comment:
@mkhorton There seemed to be a lot of unneeded ignores in here. Let me know if I removed anything that was important.

Reply:
I think this was from a standard .gitignore template from somewhere, but I'm fine removing